lib/inc/cpp-pcp-client/connector/connector.hpp

```diff
@@ -154,8 +159,14 @@ class LIBCPP_PCP_CLIENT_EXPORT Connector {
     ///
     /// Throws a connection_not_init_error in case the connection has
     /// not been opened previously.
+    /// Also rethrows any exception that can be thrown by
+    /// startMonitoring.
```

Comment:
I changed my mind. Current names are good enough.

But the above docstring is false; in case the monitor task did finish due to a non-filtered exception, `stopMonitorTaskAndWait()` won't be called, thus the exception won't be re-thrown.